roles/kubernetes-apps/ingress_controller/cert_manager/templates/cert-manager-certificate-crd.yml.j2

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: certificates.certmanager.k8s.io
+  labels:
+    app: cert-manager
+    chart: cert-manager-0.2.5
```

Comment:
static version id?

Reply:
I keep this from https://github.com/jetstack/cert-manager/tree/master/docs/deploy/rbac since it could simplify the upgrade of this port, when we need to do diff and apply with upstream changes.